README.md

```diff
@@ -168,3 +168,9 @@ This allows you to locally test the application does an actual OCR image to text
 ``` bash
 mvn test -Dincluded.tests=integration-test
 ```
+
+Note - If using Docker for local development, e.g. `docker_chs up` or `./chs-dev up`, then you need to run
+```bash
+export OCR_TESSERACT_POC_URL=http://api.chs.local/
```

Comment:
do we need to export this manually or could we just have it in the docker-compose file in dockerCHS?

Reply:
i tried adding `OCR_TESSERACT_POC_URL=http://api.chs.local/` to the `ocr-api.docker-compose.yaml` in `environment:` but this doesn't seem to work. I think this sets env vars for the running container maybe.